cluster-state-service/cmd/root.go

```diff
@@ -46,6 +47,7 @@ a localized data store, which provides you a near-real-time view of your cluster
 	}
 	// TODO: Fix the description
 	rootCmd.PersistentFlags().StringVar(&config.SQSQueueName, sqsQueueNameFlag, "", "SQS queue name")
+	rootCmd.PersistentFlags().StringVar(&config.KinesisStreamName, kinesisStreamNameFlag, "", "Kinesis stream name")
```

Comment:
As in interface decision, I'm not sure I like adding `--stream` as this may proliferate flag names as we consider additional facilities for acquiring events and the distinction between `queue` and `stream` is muddy at best. Perhaps a `--queue-type` which can have values of `sqs` and `kinesis`, defaulting to `sqs` to preserve prior behavior?